processflow.md

```diff
@@ -1,24 +1,30 @@
-# Processing Flows
+# Processing flows
 
-This diagram depicts the "happy-path" flow through the [HRI](glossary.md#hri)  for a single [batch](glossary.md#batch). 
+Figure 1 provides an overview of how processing works in the Health Record Ingestion service. In it, you'll see a normally-functioning flow through Health Record Ingestion for a single [batch](glossary.md#batch). Each numbered step in the diagram is described below.
```

Comment:
"happy-path" is actually a general use case term (https://en.wikipedia.org/wiki/Happy_path), and I don't think 'normally-functioning flow' is. The figure does not really provide an overview of how processing works. It's limited to one particular use case flow.